src/components/views/rooms/RoomSettings.js

```diff
@@ -602,6 +611,7 @@ module.exports = React.createClass({
                           level={SettingLevel.ROOM_DEVICE}
                           roomId={this.props.room.roomId}
                           manualSave={true}
+                          disabled={!this.refs.encrypt || !this.refs.encrypt.checked}
```

Comment:
Wondering if ref will be available here on the first render ... ? Why not use `isEncrypted` here?

Reply:
actually, this doesn't work as isEncrypted is the current, not proposed state of the room.